app/smc/prj.conf

```diff
@@ -26,3 +26,6 @@ CONFIG_BOOT_BANNER=n
 CONFIG_TT_BOOT_BANNER=y
 CONFIG_TT_BOOT_BANNER_GIT_VERSION=y
 CONFIG_TT_BOOT_BANNER_SDK_VERSION=y
+
+#CRC
+CONFIG_CRC=y
```

Comment:
Can probably drop these here and just make `bh_arc` `select CRC` here
https://github.com/tenstorrent/tt-zephyr-platforms/blob/4c4c6ce2111c2080181e3b33223d9e7762ecef3f/lib/tenstorrent/bh_arc/Kconfig#L9